src/uct/ib/ud/base/ud_ep.h

```diff
@@ -238,6 +238,7 @@ struct uct_ud_ep {
     ucs_wtimer_t     slow_timer;
     uint8_t          flags;
     uint8_t          path_bits;
+    ucs_queue_elem_t fail_queue;
```

Comment:
pls check if it is possible to use union and combine fail_queue with other ep members ?